modules/core/src/main/scala/fs2/kafka/KafkaProducer.scala

```diff
@@ -82,17 +82,15 @@ object KafkaProducer {
             override def produce[P](
               records: ProducerRecords[K, V, P]
             ): F[F[ProducerResult[K, V, P]]] = {
-              withProducer { producer =>
+              withProducer { (producer, _) =>
                 records.records
                   .traverse(produceRecord(keySerializer, valueSerializer, producer))
                   .map(_.sequence.map(ProducerResult(_, records.passthrough)))
               }
             }
```

Comment:
With this change, this no longer shifts to the blocking pool - am I right in changing this, since it's invoking `produceRecord` which is async? (@SystemFw )

Reply:
yeah, that's right